scripts/dev/run-docker-compose.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# A wrapper script to run Docker Compose commands
+
+# Reverse condition: just exit if docker not installed
+if ! command -v docker-compose &>/dev/null; then
+  echo "Docker Compose is not installed. See https://docs.docker.com/compose/install/."
+  exit 1
+fi
+
+if docker-compose version &>/dev/null; then
+  docker-compose "$@"
+else
+  echo "Docker Compose requires root privileges. Running with sudo: docker $@"
+  sudo docker-compose "$@"
+fi
```

Comment:
The root issue is that some people choose to install `docker-compose` as a standalone command, while others like me install it as a plugin (`docker compose`) - see https://docs.docker.com/compose/install/. Would be great if the script could support the two:
1. Check if `docker-compose` is installed (`which docker-compose`)
2. If the command exists, run it. Retry with root if necessary.
3. If not existing, check if `docker compose` is supported. If so, run it with `./run-docker.sh`
4. If none of this is supported, show the install error message.